subsys/bluetooth/mesh/cfg_srv.c

```diff
@@ -2072,6 +2072,9 @@ static void lpn_timeout_get(struct bt_mesh_model *model,
 		goto send_rsp;
 	}
 
+	/* This looks really wrong: the variable says the value is in
+	 * ms, but the assigned value is ms / 100.
+	 */
 	timeout_ms = k_delayed_work_remaining_get(&frnd->timer) / 100;
```

Comment:
Based on the Mesh spec "The PollTimeout timer value in units of 100 milliseconds", i.e. the variable name is wrong here but the code does the right thing.